CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+Welcome to the GitHub page of the Swiss NextGen API. Please open an Issue if you want to suggest new features or submit adjustments (ex. fixing a typo), you can also submit a Pull-Request directly.
+
+## Regular Updates to the API
+
+- Berlin Group XS2A API updates: As soon as possible after new releases
+- Six Payments Definitions: Yearly
+
+## Rules
+
+- Adjustments to the API can be submitted as an Issue or directly as a pull request
+- Outstanding Adjustments will be reviewed and prioritized regularly by the admins
+- Adjustments must be commited to the develop branch
```

Comment:
I'm not so sure that this is what we want.  Either we have a development branch which we merge into the master branch from time to time. In this case an adjustment must be made on a feature or bugfix branch and a PR is submitted to be merged into the development branch.
The other case is that we work on the master. Then we don't need the develop branch anymore (can be deleted) and adjustments are made on a feature or bugfix branch and a PR is submitted to be merged into the master branch.

Reply:
I am for the first variant because then we have a master branch reflecting always the latest state of the API and any change on the master will lead to a new release.

Reply:
Ok @wishopt , please write:

- Adjustments must be committed to a feature or bugfix branch (named _feature/\<name\>_ or _bugfix/\<name\>_) followed by a pull request to master.

Reply:
hmm did you mean "..followed by a pull request to develop"  ? Or do you want to get rid of the develop branch?

Reply:
Ooops, I meant 'develop'. Thank you. I edited my original comment.